src/planner/macos/mod.rs

```diff
@@ -72,6 +72,20 @@ pub struct Macos {
     /// The root disk of the target
     #[cfg_attr(feature = "cli", clap(long, env = "NIX_INSTALLER_ROOT_DISK"))]
     pub root_disk: Option<String>,
+
+    /// On AWS, put the Nix Store volume on the EC2 instances' instance store volume.
+    ///
+    /// Using the instance store volume bypasses the interactive "enable full disk access" step.
+    /// However, using the instance store volume means the machine can never be "Stopped".
+    /// If the instance is Stopped, the instance store volume is erased, and the installation is broken.
+    /// Note: the machine can still be rebooted.
+    ///
+    /// Setting this option:
+    ///
+    ///  * Requires passing --determinate due to complications of AWS's deployment of macOS.
```

Comment:
can't we make clap require other settings? that might be useful to enforce this constraint